src/place-detail/ui/place-detail.html

```diff
@@ -29,6 +29,9 @@ <h1 class="title" ng-bind="::place.name"></h1>
                 class="list-group-item"
                 type="button"
                 ng-click="openTranslatePage()"><i class="fa fa-globe" aria-hidden="true"></i>  <span translate-once="preview.translate"></span></button>
+        <a ng-if="::permissions.editing"
+           class="list-group-item"
+           ng-href="{{ place.url + '/status' }}"><i class="far fa-calendar-check" aria-hidden="true"></i>  <span translate-once="preview.change_status"></span></a>
```

Comment:
Is it normal the class is `far` ? Or is it a typo?

Reply:
I think that's the class name for the regular icon set. https://fontawesome.com/how-to-use/on-the-web/referencing-icons/basic-use